### DISCUSSION TITLE
Multiplatform app doesn't work on MacOS

### DISCUSSION THREAD

**Discussion by EricFordConsulting** [pointfreeco/swift-composable-architecture#1066] [Q&A]
It must be something to do with the project configuration.

I'm attaching the failing project: TCACatalystTest

And a MacOS only project that works fine: MacOSTCATest
[TCACatalystTest.zip](https://github.com/pointfreeco/swift-composable-architecture/files/8577211/TCACatalystTest.zip)
[MacOSTCATest.zip](https://github.com/pointfreeco/swift-composable-architecture/files/8577212/MacOSTCATest.zip)

**Comment by haikusw** [comment]
Took a look.  Had to add "ComposableArchitecture" binary to the "Link Binary With Libraries" Build Phase for the iOS catalyst target to get it to build and link.  Runs in iOS 15.4 simulator and on macOS 12.3.1 once I do that.

![Screen Shot 2022-04-29 at 12 26 11 PM](https://user-images.githubusercontent.com/222271/166056086-22adab2e-e846-4f49-b4a3-05b5844cfeff.png)
.

**Reply by EricFordConsulting** [reply]
Worked like a charm

**Reply by haikusw** [reply]
Great!  Guess this issue can get closed out then.

**Comment by EricFordConsulting** [comment]
For sure. Was I supposed to close it?

On Sat, May 7, 2022 at 1:37 PM Tyler ***@***.***> wrote:

> Great! Guess this issue can get closed out then.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pointfreeco/swift-composable-architecture/discussions/1066#discussioncomment-2707327>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHDCC6AJNUW6DLAJBACTSJLVI3H73ANCNFSM5UQGAJJA>
> .
> You are receiving this because you authored the thread.Message ID:
> <pointfreeco/swift-composable-architecture/repo-discussions/1066/comments/2707327
> @github.com>
>


-- 
--------------
Eric Ford
(707) 787-8082
***@***.***

**Reply by haikusw** [reply]
Hmm.. I thought it was an "Issue".   Not as familiar with GH  "Discussion" items.
It does still say "Unanswered" at the top…  Ah yes, it looks like there should be a button on my answer - **"Mark as answer"** that will set this item as "answered" so that people searching for unanswered questions to help with won't have this show up as noise

![Screen Shot 2022-05-07 at 3 46 51 PM](https://user-images.githubusercontent.com/222271/167274304-fde800ed-bbfb-4bcd-b883-872f085d9d3c.png)
.
